en-US/emails/2018/firefox_learning_journey.lang

```diff
@@ -0,0 +1,308 @@
+## NOTE: Firefox learning journey through a series of emails, each with a theme.
+## NOTE: Bug 1458771; reference the links to mockup images for context.
+
+
+# Email: General mobile
+# Mockup image: https://goo.gl/nn9WPh
+# Subject
+;Take Firefox with you
+Take Firefox with you
+
+
+# Pre-header
+;Wherever you roam
+Wherever you roam
+
+
+# Headline
+;Roam free on the open web
+Roam free on the open web
+
+
+;Browse like no one's watching, no matter where you are. Firefox sets you — and your phone — free online.
+Browse like no one's watching, no matter where you are. Firefox sets you — and your phone — free online.
+
+
+# CTA
+;App Store Badges
+App Store Badges
+
+
+# Email: Essential extensions
+# Mockup image: https://goo.gl/2pvoHr
+# Subject
+;Give your browser superpowers with extensions
+Give your browser superpowers with extensions
+
+
+# Pre-header
+;Discover the top essential extensions to customize Firefox.
+Discover the top essential extensions to customize Firefox.
+
+
+# Headline
+;Discover Essential Extensions for Firefox
+Discover Essential Extensions for Firefox
+
+
+# Toppings on a baked potato are meant to enhance the flavor of a potato. Here it compares extensions with the toppings on a baked potato, they give a product feature a boost. Feel free to replace it with cultural relevant expression.
+;Extensions are like toppings for your baked potato — once you try them, you can't imagine going without them. Firefoxers use extensions to give their privacy a boost, play games, block ads, manage video downloads and all kinds of other extraordinary stuff.
+Extensions are like toppings for your baked potato — once you try them, you can't imagine going without them. Firefoxers use extensions to give their privacy a boost, play games, block ads, manage video downloads and all kinds of other extraordinary stuff.
+
+
+;Give your browser superpowers and discover the top Firefox extensions:
+Give your browser superpowers and discover the top Firefox extensions:
+
+
+# CTA
+;See the top extensions
+See the top extensions
+
+
+# Email: Pocket
+# Mockup image: https://goo.gl/57ZQoR
```

Comment:
The image in the doc is quite confusing… Are they going to replace the huge graphic in the header?